tajo-catalog/tajo-catalog-client/src/main/java/org/apache/tajo/catalog/AbstractCatalogClient.java

```diff
@@ -430,27 +454,32 @@ public final PartitionDescProto getPartition(final String databaseName, final St
   }
 
   @Override
-  public boolean addPartitions(String databaseName, String tableName, List<PartitionDescProto> partitions
-    , boolean ifNotExists) {
+  public void addPartitions(String databaseName, String tableName, List<PartitionDescProto> partitions,
+                               boolean ifNotExists) throws UndefinedTableException, DuplicatePartitionException,
+      UndefinedPartitionMethodException {
```

Comment:
```UndefinedDatabaseException``` also should be handled. 
In addition, ```DuplicatePartitionException``` is missed at CatalogServer.addPartitions(). It would be great if you add it.